packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -25,10 +25,6 @@ import {juggler, Repository} from '../repositories';
 
 const debug = debugFactory('loopback:repository:mixin');
 
-// FIXME(rfeng): Workaround for https://github.com/microsoft/rushstack/pull/1867
-/* eslint-disable @typescript-eslint/no-unused-vars */
-import * as loopbackContext from '@loopback/core';
-import * as loopbackCore from '@loopback/core';
```

Comment:
Please revert the changes above. IDE tooling is sorting imports in a way that breaks our API doc generator, we have to manually preserve the explicit order.

Reply:
i didn't get your point, i don't see those imports

Reply:
i solved it, i didn't see that it was removed